modules/core/src/main/scala/org/scalasteward/core/forge/gitlab/GitLabApiAlg.scala

```diff
@@ -61,6 +63,7 @@ private[gitlab] object MergeRequestPayload {
       reviewer_ids = Option.when(reviewers.nonEmpty)(reviewers),
       labels = Option.when(data.labels.nonEmpty)(data.labels),
       target_project_id = projectId,
+      remove_source_branch = Option.when(removeSourceBranch)(removeSourceBranch),
```

Comment:
I am using _flag_ to read `removeSourceBranch` config we can't actually pass `remove_source_branch = false` but I guess that's ok. Alternative is to make CLI option `Option[Boolean]` something like this;

```scala
implicit val readBoolean: Argument[Boolean] =
  Argument.from("boolean") { string =>
    try
      Validated.valid(string.toBoolean)
    catch {
      case _: IllegalArgumentException => Validated.invalidNel(s"Invalid Boolean: $string")
    }
  }

private val gitlabRemoveSourceBranch_v2: Opts[Option[Boolean]] =
  option[Boolean](
    "gitlab-remove-source-branch",
    "Flag indicating if a merge request should remove the source branch when merging."
  ).orNone

```

Let me know what you think.